lib/archethic/contracts/interpreter/library.ex

```diff
@@ -217,4 +195,33 @@ defmodule Archethic.Contracts.Interpreter.Library do
       bin
     end
   end
+
+  @doc """
+  Get the genesis address of the chain
+  """
+  @spec get_genesis_address(binary()) ::
+          binary()
+  def get_genesis_address(address) do
+    addr_bin = decode_binary(address)
+
+    case TransactionChain.fetch_genesis_address_remotely(addr_bin) do
```

Comment:
Why creating a new `TransactionChain.fetch_genesis_address_remotely` without the node list ?

Reply:
There is already a function `fetch_genesis_address_remotely/2` why not reusing this one ?